app/src/displays/formatted-json-value/formatted-json-value.vue

```diff
@@ -1,19 +1,38 @@
 <template>
 	<value-null v-if="!displayValue" />
+	<v-menu v-else-if="displayValue.length > 1" show-arrow>
+		<template #activator="{ toggle }">
+			<span @click.stop="toggle" class="toggle">
+				<span class="label">
+					{{ displayValue.length }}
+					<template v-if="displayValue.length >= 100">+</template>
```

Comment:
We don't need the "if 100, show +" logic here, as we always know the full count. The 100+ is from the o2m display that can only get an accurate count between 0 and 100 items